TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/driveTrain/commands/RotateRobotByDegCommand.java

```diff
@@ -1,13 +1,13 @@
 package org.firstinspires.ftc.teamcode.subsystems.driveTrain.commands;
 
+import com.arcrobotics.ftclib.command.CommandBase;
 import com.arcrobotics.ftclib.controller.PController;
 
 import org.firstinspires.ftc.teamcode.subsystems.driveTrain.IDriveTrainSubsystem;
 import org.firstinspires.ftc.teamcode.util.DataLogger;
-import org.firstinspires.ftc.teamcode.util.subsystems.SympleCommandBase;
 
-public class RotateRobotByDegCommand extends SympleCommandBase<IDriveTrainSubsystem> {
-    public static final double defaultKp = 0.05f;
+public class RotateRobotByDegCommand extends CommandBase {
+    public static final double DEFAULT_KP = 0.05f;
```

Comment:
Why Add f postfix if you use a double instead of a float.
It just triggers me so MAKOT!!

Reply:
fixed